gcp_variant_transforms/testing/integration/huge_tests/test_1000_genomes.json

```diff
@@ -0,0 +1,23 @@
+{
+  "test_name": "test-1000-genomes-no-merge",
+  "table_name": "test_1000_genomes_no_merge",
+  "input_pattern": "gs://genomics-public-data/1000-genomes/vcf/ALL*phase1*vcf",
```

Comment:
Question: IIUC this includes all files in the vcf directory except `ALL.chrMT.phase1_samtools_si.20101123.snps.low_coverage.genotypes.vcf`; is there a particular reason to exclude this single file?

Reply:
Just now saw the reference to Issue #61 in your PR description; is that the reason for exclusion of this file?